internal/mapping.go

```diff
@@ -144,27 +144,44 @@ func GetSpannerCols(conv *Conv, srcTable string, srcCols []string) ([]string, er
 // (across the database). But in some source databases, such as PostgreSQL,
 // they only have to be unique for a table. Hence we must map each source
 // constraint name to a unique spanner constraint name.
```

Comment:
As discussed in the group chat, the name returned here must not conflict with a Spanner table name.

What I suggest is that we just add a TODO for this PR so we can get it in. I think the likelihood of mysql and postgres databases using the same name for tables, indexes or foreign key constraints is fairly small. We’ll handle it with a follow up PR where we can focus on doing the refactoring.

Suggest adding a TODO:

// TODO: ensure that the name we return does not conflict with a Spanner table name.

Reply:
Added a quick fix and a background comment on it. we will pre populate usedNames with Spanner table names to handle collision with foreign key constraint name and index name.